### PR TITLE
Update meeting protocol: add presidency and secretary role

### LINKED ISSUE #1655: Rolle auch für Vorsitz/Protokollführung darstellen
Im Moment wird im Protokoll nur die Rolle für Mitglieder dargestellt (In Klammern hinter dem Namen). Dies sollte auch bei Vorsitz/Protokollführung passieren.

### PR DESCRIPTION
**Pull request by phgross**
- Also provide the role for the meetings presidency and secretary in the JSON data.
- Add presidency and secretary role to the example protocol sablon template.
- Harmonize format of all participants (members, presidency, secretary).

Note: Because of the harmonization the former keys `participants.presidency` and `participants.secretary` are no longer available, therefore the Sablontemplates has to be update after installing this change on production!

Fixes #1655.

bildschirmfoto 2016-04-12 um 11 03 51

@deiferni

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,13 @@ Changelog
 4.7.5 (unreleased)
 ------------------
 
+- Meeting Protocol:
```
Comment (lukasgraf) [inline docs/HISTORY.txt]: I think this nested list needs some additional newlines to be valid restructured text - see other nested lists in the changelog.

**Comment by lukasgraf** [comment]
Can we get tests for this?

**Comment by phgross** [comment]
@lukasgraf it's already tested in the `TestProtocolJsonData` tests, thats the reason why I've changed the `SAMPLE_MEETING_DATA`in the `sablontemplate.py`.

I think the `def test_protocol_json` test is enough.

**Comment by lukasgraf** [comment]
Ah, now I see :wink: :+1:

File: docs/HISTORY.txt (hunk 1 above)
Comment (phgross) [inline docs/HISTORY.txt]: changelog fixed ...

File: docs/HISTORY.txt [hunk 2]
```diff
@@ -5,6 +5,15 @@ Changelog
...
+
+  - Also provide the role for the meetings presidency and secretary in the JSON data.
+  - Add presidency and secretary role to the example protocol sablon template.
+  - Harmonize format of all participants (members, presidency, secretary).
+
+  Note: The Sablontemplates has to be update after Installing this release!
```
Comment (deiferni) [inline docs/HISTORY.txt]: The Sablontemplates ha**ve** to be update**d** after ~~I~~**i**nstalling this release!

**Comment by deiferni** [comment]
Looks :+1:

**Comment by lukasgraf** [comment]
@phgross rebase-zonk :stuck_out_tongue_closed_eyes:

**Comment by phgross** [comment]
@lukasgraf or @deiferni changelog fixed and rebased ...

**Comment by deiferni** [comment]
@phgross sorry i noticed just now: currently there is a duplicate of the sablon templates in our policy template package. that one should be updated as well. 

_side-quest: (recommended) you could also remove the duplicates and copy the templates from the examplecontent profile directory on policy creation. maybe as a separate PR though._

**Comment by phgross** [comment]
@deif I've updated also the template in the `policytemplates` and created an separate issue #1717 for the planed change in policytemplates

**Comment by deiferni** [comment]
:+1:

**Pull request merged**